<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>E9_KK_TAHSILAT_BATCH</class>
 <widget class="QDialog" name="E9_KK_TAHSILAT_BATCH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>210</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>210</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="15,5,5" columnstretch="30,5">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,10,2,2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Pos Numarası</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QSearchEdit" name="searchEdit_pos_numarasi">
       <property name="whatsThis">
        <string>Arama widget. </string>
       </property>
       <property name="QSearchEdit" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEdit_doviz_cinsi">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Müşteri Kredi Kartı Numarası</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_kk_numarasi">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Kredi Kartı Sahibi</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_kk_sahibi"/>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>37</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>417</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="widget_batch_buttons" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSearchEdit</class>
   <extends>QWidget</extends>
   <header>QSearchEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>searchEdit_pos_numarasi</tabstop>
  <tabstop>lineEdit_doviz_cinsi</tabstop>
  <tabstop>lineEdit_kk_numarasi</tabstop>
  <tabstop>lineEdit_kk_sahibi</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
